qpmodel/PhysicalNode.cs

```diff
@@ -173,7 +173,21 @@ public double InclusiveCost()
                 else
                     incCost += x.InclusiveCost();
             });
-
+            //FIXED  @issue72 here should add the cost of subquery
+            if (this.logic_.filter_ != null)
+            {
+                Expr expr = this.logic_.filter_;
+                expr.VisitEachT<SubqueryExpr>(x =>
+                {
+                    if (x.query_.physicPlan_ != null)
```

Comment:
Do you have a test case x.query_.physicPlan_ is null? That's a bug shall be fixed (you can open an issue and fix separately).

Reply:
OK. And the new issue is #230